packages/cli/src/zosfiles/delete/vsam/Vsam.handler.ts

```diff
@@ -18,10 +18,26 @@ import { ZosFilesBaseHandler } from "../../ZosFilesBase.handler";
  */
 export default class VsamHandler extends ZosFilesBaseHandler {
     public async processWithSession(commandParameters: IHandlerParameters, session: AbstractSession): Promise<IZosFilesResponse> {
-        return Delete.vsam(session, commandParameters.arguments.dataSetName, {
-            erase: commandParameters.arguments.erase,
-            purge: commandParameters.arguments.purge,
-            responseTimeout: commandParameters.arguments.responseTimeout
-        });
+        try {
+            const response = await Delete.vsam(session, commandParameters.arguments.dataSetName, {
+                erase: commandParameters.arguments.erase,
+                purge: commandParameters.arguments.purge,
+                responseTimeout: commandParameters.arguments.responseTimeout
+            });
+
+            return response;
+        } catch (error: any) {
+            if (commandParameters.arguments.ignoreNotFound &&
+                (error.errorCode === '404' || error.toString().includes("IDC3012I"))) {
+                return { success: true, commandResponse: "VSAM dataset not found but this is ignored" };
+            }
+            if (error instanceof ImperativeError){
+                throw error;
+            }
+            throw new ImperativeError({
+                msg: error.mMessage || "An unexpected error occurred.",
```

Comment:
The `mMessage` property is mostly likely undefined if the error is not an instance of `ImperativeError`. Should we do the following to match the base handler?
```suggestion
                msg: error.message,
```